templates/modify-vm-template/manifests/modify-vm-template.yaml

```diff
@@ -0,0 +1,87 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: ClusterTask
+metadata:
+  annotations:
+    task.kubevirt.io/associatedServiceAccount: {{ sa_name }}
+    templateName.params.task.kubevirt.io/kind: {{ task_param_types.template_kind }}
+    templateName.params.task.kubevirt.io/apiVersion: {{ task_param_types.template_version }}
+    templateNamespace.params.task.kubevirt.io/kind: {{ task_param_types.namespace }}
+    cpuSockets.params.task.kubevirt.io/type: {{ task_param_types.string }}
+    cpuCores.params.task.kubevirt.io/type: {{ task_param_types.string }}
+    cpuThreads.params.task.kubevirt.io/type: {{ task_param_types.string }}
+    memory.params.task.kubevirt.io/type: {{ task_param_types.string }}
+  labels:
+    task.kubevirt.io/type: {{ task_name }}
+    task.kubevirt.io/category: {{ task_category }}
+  name: {{ task_name }}
+spec:
+  params:
+    - name: templateName
+      description: Name of an OpenShift template.
+      type: string
+    - name: templateNamespace
+      description: Namespace of an source OpenShift template. (defaults to active namespace)
+      type: string
+    - name: cpuSockets
+      description: Number of CPU sockets
+      type: string
+    - name: cpuCores
+      description: Number of CPU cores
+      type: string
+    - name: cpuThreads
+      description: Number of CPU thready
+      type: string
+    - name: memory
+      description: Number of memory vm can use
+      type: string
+    - description: Template labels. If template contains same label, it will be replaced.
```

Comment:
we should specify a format of these: something akin to https://github.com/kubevirt/kubevirt-tekton-tasks/blob/main/tasks/create-vm-from-template/manifests/create-vm-from-template.yaml#L35